lib/common/ipc_controld.c

```diff
@@ -362,8 +362,8 @@ pcmk_controld_api_reprobe(pcmk_ipc_api_t *api, const char *target_node,
         router_node = target_node;
     }
     crm_debug("Sending %s IPC request to reprobe %s via %s",
-              pcmk_ipc_name(api, true), crm_str(target_node),
-              crm_str(router_node));
+              pcmk_ipc_name(api, true), pcmk__s(target_node, "unknown node"),
+              pcmk__s(router_node, "unknown node"));
```

Comment:
in cases like this it might be possible to drop the whole 'via' part if no route-node is given.
just don't know how a generic convenience-function would look like. maybe a format-string
as parameter and return a thread-variable.

Reply:
Well this one is easier ... for reprobes a NULL target or router means the local node. We could drop the "via" clause if target and router are the same.